lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -427,6 +427,7 @@ protected function transformSignup($data, $current) {
     return [
       'id' => $data->id,
       'created_at' => $data->created_at,
+      'northstar_id' => $data->northstar_id,
```

Comment:
So this isn't necessarily a data property that should be on the signup object. Placing it here for convenience is a bad precedent to set. I would say that this `northstar_id` should be a property of the user object and not of the signup. The `user` object can be nested into the `signup` object in the `SignupTransformer`, which would make more sense since the signup is associated with a particular user. Plus, then the NS id would be a regular property on the user object.